src/test/regress/expected/multi_subquery_complex_reference_clause.out

```diff
@@ -349,7 +349,7 @@ ON user_buy_test_table.item_id = users_ref_test_table.id;
      4
 (1 row)
 
--- subquery without FROM cannot be the outer relationship in an outer join
+-- subquery without FROM cannot be the outer relationship in an outer join before pg12
```

Comment:
Adding an extra test output for just one test seems aggressive to me. The issue is that we should change  `SELECT 5 AS id` such that it is also qualified as `HasEmptyJoinTree()`. If you can do that, this test should not change between the versions.

Reply:
I noted on the e-mail as well. It turns out that the test outputs are the same between pg 11 and pg 12. Can you double check please? If so, lets remove _2.out